app/konnect/getting-started/add-api.md

```diff
@@ -20,7 +20,9 @@ In this guide, you will build on your Konnect environment by adding your own API
 
 ## Prerequisites
 
-If you didn't use the in-app onboarding wizard in {{site.konnect_short_name}}, open the Konnect [Gateway Manager](https://cloud.konghq.com/gateway-manager), and create a new data plane node.
+* [A {{site.konnect_short_name}} account](https://konghq.com/products/kong-konnect/register)
+* If you didn't use the in-app onboarding wizard in {{site.konnect_short_name}}, open the Konnect [Gateway Manager](https://cloud.konghq.com/gateway-manager), and create a new data plane node.
+* If you did use the in-app onboarding wizard, make sure to delete the examples you created as part of the wizard. This will help you conserve your free trial credits in {{site.konnect_short_name}}.
```

Comment:
@connorckong I mentioned deleting the examples to save free trial credits here and in the important note below on the same page. These were the only two spots that made sense (to me) to include that message since we don't talk about the onboarding wizard workflow on another page. It's mentioned on the get started overview page, but that isn't a good spot for a note about deleting the demo.

Reply:
I think the best spot to have a note about deleting the demo would be in the UI, maybe after you're done with onboarding.

Reply:
Given this is not a true prereq for completing the steps in this doc, users might only come across this advice after depleting their trial credits, assuming they don't overlook the prereq section altogether. 

If we anticipate that this could be a problem, this information should exist directly within the onboarding workflow so that it is seen in the most relevant moment, and so that we address a users needs as part of the core onboarding wizard experience.